### ISSUE TITLE
The script generated inline with the command {% js_reverse_inline %} generates and error on Chrome

### ISSUE THREAD

**Issue by vladlep** [ierror/django-js-reverse#32]
Hi,

I am using django-js-reverse to get in javascript my urls and it works great but i needed to change the importing so the file is not cached on the client side after server restart. The website supports 2 languages 'en' and 'nl'. I am trying to figure out what is the best way to import the script since i am also using django-compressor.

Now i tried to use it as template tag https://github.com/ierror/django-js-reverse#usage-as-template-tag
but in Chrome i get an error. This is logged in the console, so for each user I would get a sentry error which  would not be ideal.

    <script type="text/javascript" charset="utf-8">
    {% js_reverse_inline %}
     </script>

This is the error:

![chrome_error](https://cloud.githubusercontent.com/assets/1110074/8328460/2c0c253a-1a71-11e5-9011-44c934d6f4f8.png)

I am also trying which is the best option to include the JS case in my case? If i cache it, i need to update it at the server restart since new url might appear. I tried to use the loading as static file but since I have 2 languages it does not work for me well (also generates the urls with en-us instead of en). Do you have other suggestions? 

Thanks,
Vlad

**Comment by mjnaderi** [comment]
I have the same issue. It generates `Content-Type: application/javascript` in output.

The problem is that in `templatetags/js_reverse.py`, the request object is passed to views.urls_js, and in views.urls_js we have this code:
```js
    if not request:
        return response_body
    else:
        return HttpResponse(response_body, **{'content_type': 'application/javascript'})
```
So instead of returning `response_body`, it returns `HttpResponse(response_body, ...)`.

**Comment by ierror** [comment]
This problem is fixed in v0.6.0 https://pypi.python.org/pypi/django-js-reverse/0.6.0 https://github.com/ierror/django-js-reverse/tree/v0.6.0
